src/Components/Navbar/Navbar.jsx

```diff
@@ -1,69 +1,70 @@
-import MobileMenu from '../MobileMenu/MobileMenu'
-import AccountMenu from '../AccountMenu/AccountMenu'
-import { useLocation, useNavigate } from 'react-router-dom'
+import React, { useCallback, useEffect, useState } from "react";
+import logo from "../../assets/netflix-logo.svg";
+import profilePic from "../../assets/BlueNetflix.jpg";
+import NavbarItem from "./NavbarItem";
+import { BsChevronDown, BsSearch, BsBell } from "react-icons/bs";
+import MobileMenu from "../MobileMenu/MobileMenu";
+import AccountMenu from "../AccountMenu/AccountMenu";
+import { useLocation, useNavigate } from "react-router-dom";
 
 const TOP_OFFSET = 66;
 
-const Navbar = () => {
+const Navbar = ({ clickStateAccountMenu, setClickStateAccountMenu }) => {
   const [showMobileMenu, setShowMobileMenu] = useState(false);
-  const [showAccountMenu, setShowAccountMenu] = useState(false);
   const [showBackground, setShowBackground] = useState(false);
   const [searchActive, setSearchActive] = useState(false);
-  const { pathname } = useLocation()
-  const [lastURL, setLastURL] = useState("")
+  const { pathname } = useLocation();
+  const [lastURL, setLastURL] = useState("");
 
   const searchHandler = () => {
     setSearchActive(!searchActive);
   };
 
   const navigate = useNavigate();
 
-  
   useEffect(() => {
     const handleScroll = () => {
-      if(window.scrollY >= TOP_OFFSET){
-        setShowBackground(true)
-      }
-      else{
+      if (window.scrollY >= TOP_OFFSET) {
+        setShowBackground(true);
+      } else {
         setShowBackground(false);
       }
-    }
-    window.addEventListener('scroll', handleScroll);
-    return() =>{
-      window.removeEventListener('scroll', handleScroll);
-    }
-  },[]);
-  
+    };
+    window.addEventListener("scroll", handleScroll);
+    return () => {
+      window.removeEventListener("scroll", handleScroll);
+    };
+  }, []);
+
   const toggleMobileMenu = useCallback(() => {
     setShowMobileMenu((current) => !current);
-  },[]);
-  
-  const toggleAccountMenu = useCallback(() => {
-    setShowAccountMenu((current) => !current);
-  },[]);
+  }, []);
+
+  const toggleAccountMenu = useCallback(
+    (e) => {
+      e.stopPropagation(); // Prevent event from bubbling up
+      setClickStateAccountMenu((current) => !current);
+    },
+    [setClickStateAccountMenu]
+  );
 
   const onChangeHandler = (e) => {
-    if(e.target.value == '')
-    {
+    if (e.target.value == "") {
       navigate(lastURL);
-      setLastURL("")
-    }
-    else{
-      if(pathname != "/search")
-      {
-        setLastURL(pathname)
+      setLastURL("");
+    } else {
+      if (pathname != "/search") {
+        setLastURL(pathname);
       }
-      navigate("/search?query=" + e.target.value)
+      navigate("/search?query=" + e.target.value);
     }
-  }
-  
-  if(pathname == '/register' || pathname == '/login' || pathname == '/content')
-  {
+  };
+
+  if (
+    pathname == "/register" ||
```

Comment:
make the exported consts in a separate file, exmple:

Paths.js
`
export const RegisterPathname = "/register";
export const LoginPathname = "/login";
export const ContentPathname = "/content";
`